apps/web/pages/api/event-type/webhook.ts

```diff
@@ -0,0 +1,86 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Shouldn't these endpoints be tRPC mutations? Since we're trying to all logic in there so we can reuse it in the upcoming open API.

Reply:
good catch. We don't need this. I made it but then moved to tRPC mutations and somehow forgot it in the mix. Removing this